superset/assets/src/components/FilterableTable/FilterableTable.jsx

```diff
@@ -29,14 +30,59 @@ import {
   Table,
 } from 'react-virtualized';
 import { getTextDimension } from '@superset-ui/dimension';
+import { t } from '@superset-ui/translation';
+
+import Button from '../Button';
+import CopyToClipboard from '../CopyToClipboard';
+import ModalTrigger from '../ModalTrigger';
 import TooltipWrapper from '../TooltipWrapper';
 
 function getTextWidth(text, font = '12px Roboto') {
   return getTextDimension({ text, style: { font } }).width;
 }
 
+function getJsonStringifiedObject(data) {
```

Comment:
NIT (optional): naming `safeJsonParse`

Reply:
I'll make it `safeJsonObjectParse` since it doesn't parse all JSON, but only things that are objects.